src/vcBPA.cpp

```diff
@@ -398,16 +405,11 @@ vcBPATriangle vcBPA_FindSeedTriangle(vcBPAGrid *pGrid, double ballRadius)
         udULong3 triangle = udULong3::create((uint64_t)pointIndex, (uint64_t)nearbyPoints[i], (uint64_t)nearbyPoints[j]);
 
         // Check that the triangle normal is consistent with the vertex normals - i.e. pointing outward
-        udDouble3 a = p1 - p0;
-        udDouble3 b = p2 - p0;
-        udDouble3 triangleNormal = udNormalize(udCross(a, b));
+        udDouble3 triangleNormal = vcBPA_GetTriangleNormal(p0, p1, p2);
         if (triangleNormal.z < 0) // Normal is pointing down, that's bad!
```

Comment:
In the interest of avoiding `triangleNormal` existing beyond this point with the wrong value, perhaps we should merge these two lines? `vcBPA_GetTriangleNormal(p0, p1, p2).z < 0`?

My main concern is that if we started using `triangleNormal` after this conditional, it would sometimes be wrong as the normal is no longer updated inside this block. I don't believe there's any reason for us to use the `triangleNormal` though.